packages/aila/src/core/Aila.testHelpers.ts

```diff
@@ -0,0 +1,69 @@
+import { expect } from "@jest/globals";
+import invariant from "tiny-invariant";
+
+import { Aila } from ".";
+import { MessagePart, TextDocument } from "../protocol/jsonPatchProtocol";
+
+export function checkAssistantResponse(content: string) {
```

Comment:
Add test helpers for live LLM tests to make sure we don't have backticked JSON etc. being returned